git_machete/cmd.py

```diff
@@ -1396,16 +1394,16 @@ def sync_annotations_to_github_prs(machete_client: MacheteClient) -> None:
     machete_client.save_definition_file()
 
 
-def is_merged_to_upstream(machete_client: MacheteClient, cli_ctxt: CommandLineContext, b: str) -> bool:
+def is_merged_to_upstream(machete_client: MacheteClient, b: str) -> bool:
     if b not in machete_client.up_branch:
         return False
-    return is_merged_to(cli_ctxt, b, machete_client.up_branch[b])
+    return is_merged_to(machete_client.cli_ctxt, b, machete_client.up_branch[b])
 
 
 # Parse and evaluate direction against current branch for show/go commands
-def parse_direction(machete_client: MacheteClient, cli_ctxt: CommandLineContext, param: str, b: str, allow_current: bool, down_pick_mode: bool) -> str:
+def parse_direction(machete_client: MacheteClient, param: str, b: str, allow_current: bool, down_pick_mode: bool) -> str:
```

Comment:
This one, in turn, probably should NOT be a part of `MacheteClient` &mdash; too much command-line input parsing involved, that won't be handy/useful for programmatic access